cmd/configs.go

```diff
@@ -100,6 +105,7 @@ func printConfig() {
 	fmt.Println("EndDate:", mc.Config.EndDate)
 	fmt.Println("NoDestroy:", mc.Config.NoDestroy)
 	fmt.Println("DisableShell:", mc.Config.DisableShell)
+	fmt.Println("DisableShell:", mc.Config.Version)
```

Comment:
We might want to put the version field near the top in our examples and helper output, so it's more apparent to the image dev?

Reply:
In case you can’t tell, that was thrown together in 5 seconds. I just noticed that the debug output still says "DisableShell". Will fix and move in a bit.

Reply:
Version would probably be best to permanently have at the top, as it's the most likely thing to need to be changed/updated.